Aztec/Classes/TextKit/MediaAttachment.swift

```diff
@@ -298,22 +298,31 @@ open class MediaAttachment: NSTextAttachment {
     /// Otherwise, we'll always take the whole container's width.
     ///
     override open func attachmentBounds(for textContainer: NSTextContainer?, proposedLineFragment lineFrag: CGRect, glyphPosition position: CGPoint, characterIndex charIndex: Int) -> CGRect {
-        
+
         ensureImageIsUpToDate(in: textContainer)
-        
+
         if image == nil {
             return .zero
         }
 
-        var padding = (textContainer?.lineFragmentPadding ?? 0)
+        var padding = (textContainer?.lineFragmentPadding ?? 0) * 2
         if let storage = textContainer?.layoutManager?.textStorage,
-           let paragraphStyle = storage.attribute(NSParagraphStyleAttributeName, at: charIndex, effectiveRange: nil) as? NSParagraphStyle {
-            padding += paragraphStyle.firstLineHeadIndent + paragraphStyle.tailIndent
+            let paragraphStyle = storage.attribute(NSParagraphStyleAttributeName, at: charIndex, effectiveRange: nil) as? NSParagraphStyle {
+            let attachmentString = storage.attributedSubstring(from: NSMakeRange(charIndex, 1)).string
+            let headIndent = storage.string.isStartOfParagraph(at: attachmentString.startIndex) ? paragraphStyle.firstLineHeadIndent : paragraphStyle.headIndent
+
+            padding += headIndent
+
+            if paragraphStyle.tailIndent > 0 {
```

Comment:
Can you explain why do we need this IF, in what scenarios the tailIdent can be positive or negative?

Reply:
Here is **Discussion** of tailIndent from apple:

> If positive, this value is the distance from the leading margin (for example, the left margin in left-to-right text). If 0 or negative, it’s the distance from the trailing margin.
>
> For example, a paragraph style designed to fit exactly in a 2-inch wide container has a head indent of 0.0 and a tail indent of 0.0. One designed to fit with a quarter-inch margin has a head indent of 0.25 and a tail indent of –0.25.

Reply:
So could we rewrite this like this: 
```padding += abs(paragraphStyle.tailIndent) + abs(headIndent)```